activerecord/test/cases/adapters/postgresql/utils_test.rb

```diff
@@ -16,7 +16,8 @@ def test_extract_schema_qualified_name
       %(schema."table_name")   => %w{schema table_name},
       %("schema"."table_name") => %w{schema table_name},
       %("even spaces".table)   => ["even spaces", "table"],
-      %(schema."table.name")   => ["schema", "table.name"]
+      %(schema."table.name")   => ["schema", "table.name"],
```

Comment:
☝🏻 this is already supported with the above syntax; does it not work for your use case?

Reply:
Hmmm not sure. Given an index created like this:

```ruby
add_index 'ml.matching_suggestions', :column, name: 'index_ml.matching_suggestions_unicity'
```

I can't remove it with the following migration:

```ruby
# Intended because the index name is not standard
remove_index 'ml.matching_suggestions', %i[document_id suggested_document_id]
```

Nor with:

```ruby
# This is where the PR comes to help
remove_index 'ml.matching_suggestions', name: 'index_ml.matching_suggestions_unicity'
```

I know it's an edge case and we can just pass raw SQL. This PR is now old and if you don't see  the added value of this I can close it without being emotional 😉

Reply:
Ah okay, it's definitely true that `add_index` and `remove_index` should agree on the meaning of their arguments.

I think I would expect both to want `'"index_ml.matching_suggestions_unicity"'` here. 🤔